packages/components/drag-handle/src/DragHandle.tsx

```diff
@@ -6,7 +6,7 @@ import React, {
   MouseEventHandler,
 } from 'react';
 import { cx } from 'emotion';
-import type { PropsWithHTMLElement } from '@contentful/f36-core';
+import type { PropsWithHTMLDivElement } from '@contentful/f36-core';
```

Comment:
```suggestion
import type { PropsWithHTMLElement } from '@contentful/f36-core';
```

Reply:
I'm curious why u didn't get an TS error here

Reply:
fixed, yeah, dunno why it didn't catch it. Maybe I need to restart. Anyway, fixed. Thx!